app/package.json

```diff
@@ -49,6 +49,7 @@
     "metro-react-native-babel-preset": "0.72.3",
     "react-test-renderer": "18.1.0",
     "redux-mock-store": "^1.5.4",
+    "redux-mock-store-await-actions": "^2.1.0",
```

Comment:
This package is deprecated, let's consider writing the functionality ourselves using something similar to the store tests and polling.